internal/docker/docker.go

```diff
@@ -191,9 +191,14 @@ func (handler *Handler) StartContainer(ctx context.Context, c config.Project, s
 		return nil, err
 	}
 
-	m, err := createMounts(c.Files, DefaultProjectPath)
-	if err != nil {
-		return nil, err
+	var m []mount.Mount
```

Comment:
What if we initialize m := []mount.Mount{} at the beginning and replace else block?